utils/needRefetchListCalculation.js

```diff
@@ -0,0 +1,25 @@
+const needRefetchListCalculation = ({
```

Comment:
Please, add tests for this method also, because it has huge impact on behaviour of your logics

Reply:
Done